packages/running/src/index.tsx

```diff
@@ -213,14 +157,11 @@ function List<M>(props: SessionProps<M>) {
  *
  * It is specialized for each based on it's props.
  */
-function Section<M>(props: SessionProps<M>) {
+function Section(props: { manager: IRunningSessions.IManager }) {
   function onShutdown() {
-    void showDialog({
-      title: `Shut Down All ${props.name} Sessions?`,
-      buttons: [
-        Dialog.cancelButton(),
-        Dialog.warnButton({ label: 'Shut Down All' })
-      ]
+    showDialog({
```

Comment:
What does `void` change in this context?

Reply:
The `void` marks that we are deliberately ignoring the returned promise (which would otherwise give an error due to strict rules we applied to avoid missing accidentally dangling promises.